### PR TITLE
 Added Magneticmediajs project to files

### PR DESCRIPTION
**Pull request by arnaudleyder**
Hello, 
This is a first submission to add Magneticmediajs project to jsdelivr. Details for the project: https://www.magneticmediajs.com
I hope I have followed guidelines as indicated. 7 files are included in the magneticmediajs/1.3.1/ in files/

Thanks
Arnaud

### CODE REVIEW

File: files/magneticmediajs/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Arnaud Leyder"
+github = "https://github.com/arnaudleyder"
```
Comment (jimaek) [inline files/magneticmediajs/info.ini]: This is the project link not the author's link. So the correct is https://github.com/arnaudleyder/magneticmediajs

Comment (arnaudleyder) [inline files/magneticmediajs/info.ini]: Yes that is correct, I will keep it in mind for a next time. I have committed changes to https://github.com/arnaudleyder/jsdelivr do I need to send a new pull request?

**Comment by jimaek** [comment]
Can you now squash your commits into 1?

**Comment by arnaudleyder** [comment]
I believe I have done so now. Thanks

**Comment by jimaek** [comment]
Thank you

**Pull request merged**